DEVELOPMENT.md

```diff
@@ -0,0 +1,95 @@
+# Combo Development Guidelines
+Thank you for seeking to contribute to Combo! Any contribution is welcome and considered. 
+
+Want to get involved? Please check the [Github project](https://github.com/operator-framework/combo/projects/1) for Combo. 
+
+Have an issue you would like fix? Feel free to raise one [here](https://github.com/operator-framework/combo/issues/new/choose)!
+
+## Requirements
+
+### Tooling
+
+| Requirement | Purpose               | macOS                |
```

Comment:
Do we want to any include any other OS other than mac install requirements e.g.? (Unsure if these packages/libraries come preinstall in linux/windows)

Reply:
That's a good call out. I'd like to get this PR through and then add an issue to add those to this file. Does that sound good to you?

Reply:
That sounds good to me. I think an issue adding on too `README.md` would also be good to add as well as combo continues to grow in complexity